tests/transactional/install_updates.pm

```diff
@@ -34,7 +47,8 @@ sub run {
     }
     add_test_repositories;
     record_info 'Updates', script_output('zypper lu');
-    trup_call 'up', timeout => 1800;
+    my $ret = trup_call 'up', timeout => 1800, proceed_on_failure => 1;
+    soft_fail_rt_scriptlet if $ret;
```

Comment:
```suggestion
    soft_fail_rt_scriptlet if ($ret != 0);
```

Shouldn't it be `$ret != 0` because it's a bash return code? And yes, this is confusing.

Reply:
It is the same any value > 0 is truthy

Reply:
Ah yes that's true. I would still be explicit to avoid the same confusion as I had :sweat_smile:

Reply:
added